proto/01-protoc-base/install.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# protoc includes support for python, but not support for grpc python, we need pip to get it
+apt-get -qq update
+apt-get -qq -y --no-install-recommends install \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+    wget \
+    unzip
+rm -rf /var/lib/apt/lists/*
+dpkg --list
+
+python3 -m pip install -q --no-cache-dir --upgrade pip
+python3 -m pip install -q --no-cache-dir --upgrade setuptools
+python3 -m pip install -q --no-cache-dir -r "${__dir}/requirements.txt"
+python3 -m pip list
+
+wget -q "https://github.com/protocolbuffers/protobuf/releases/download/v3.15.6/protoc-3.15.6-linux-x86_64.zip"
```

Comment:
protoc versions probably should be made consistent between here and package lock

Reply:
Done. Should also put note or enforce deephaven-core side that it matches up; but that's for follow-up.

After updating protoc, there *were* some changes to the generated code, but only some generated javadoc comments:

```
diff --color -r /tmp/protoc-original/java/io/deephaven/proto/backplane/script/grpc/FigureDescriptorOrBuilder.java /tmp/protoc-new/java/io/deephaven/proto/backplane/script/grpc/FigureDescriptorOrBuilder.java
11c11
<    * <code>string title = 1;</code>
---
>    * <code>optional string title = 1;</code>
16c16
<    * <code>string title = 1;</code>
---
>    * <code>optional string title = 1;</code>
21c21
<    * <code>string title = 1;</code>
---
>    * <code>optional string title = 1;</code>
```